JCB_Cinema.Application/Servicies/AppUserService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AutoMapper;
+using JCB_Cinema.Application.DTOs;
+using JCB_Cinema.Application.Interfaces;
+using JCB_Cinema.Application.Requests;
+using JCB_Cinema.Domain.Entities;
+using JCB_Cinema.Domain.Interface;
+using JCB_Cinema.Infrastructure.Data.Interfaces;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace JCB_Cinema.Application.Servicies
+{
+
+    public class AppUserService : ServiceBase, IAppUserService
+    {
+        public AppUserService(IUnitOfWork unitOfWork, IMapper mapper, UserManager<AppUser> userManager, IUserContextService userContextService) : base(unitOfWork, mapper, userManager, userContextService) { }
+
+
+        public async Task<AppUserDTO?> GetAppUserAsync(string id)
+        {
+            var currentUserName = _userContextService.GetUserName();
+
+            if (string.IsNullOrEmpty(currentUserName))
```

Comment:
dodaj jeszcze warunek, ktory sprawdza czy aktualny uzytkownik ma admina, ponieważ tylko admin moze pobrac dane innego użytkownika. Użyj do tego metod z userContextService.
W momencie gdy użytkownik ma role admin to puść dalej.
Jak nie ma takiej roli to -> sprawdz czy id jest null, jak tak to pobierz dane.